docs/source/whats_new.rst

```diff
@@ -1,3 +1,54 @@
+What's New in cartopy 0.15
+==========================
+
+:Release: 0.15.0
+:Date: 1st February 2017
+
+Features
+--------
+
+* The :class:`cartopy.crs.Mercator` class now allows a ``latitude_true_scale``
+  to be specified. 
+
+* A ``tiles`` url can now be passed directly to the
+  :class:`cartopy.io.img_tiles.GoogleTiles` class. 
+
+* The :meth:`~cartopy.mpl.geoaxes.GeoAxes.background_img` method has been
```

Comment:
i'd like a bit more text about this one in the what's new.  #815 is quite a useful addition, enabling caching of prepared images, which can have large performance benefits in many cases

from the doc string:

```
Adds a background image to the map, from a selection of pre-prepared images held in a directory specified by the CARTOPY_USER_BACKGROUNDS environment variable.
```

could we grab text from this docstring or something similar to give more context to this change please?